src/error.rs

```diff
@@ -15,18 +15,23 @@ use termcolor::Buffer;
 pub type Src = String;
 pub type Name = String;
 
+#[allow(clippy::expect_used)]
```

Comment:
This function (fatal_compiler_bug), is used by gleam_expect, so I used regular expect so that we wouldn't have an infinite loop if there was a repeated bug in Buffer.